src/test/java/com/egov/core-services/constants/user.yaml

```diff
@@ -23,3 +23,65 @@ errormessages:
  invalidName: InvalidUserCreateException
  invalidTenantId: InvalidUserCreateException
```

Comment:
I think this error is common across all end points. You can define in common place and make sure you are making use of that  rather than adding it every time.

Reply:
@pundaleek-egov This change is not part of my script. So, I am not going to make any changes on this as I am not aware of the effect of this change